pyproject.toml

```diff
@@ -1,12 +1,50 @@
-[tool.coverage.run]
-omit = [
-    "viadot/flows/azure_sql_transform_dbt.py",
-    "viadot/flows/supermetrics_to_adls.py",
-    "viadot/flows/supermetrics_to_azure_sql_v2.py"
-    ]
-
-[tool.pytest.ini_options]
-log_format = "%(asctime)s %(levelname)s %(message)s"
-log_date_format = "%Y-%m-%d %H:%M:%S"
-log_cli=true
-log_level="WARNING"
\ No newline at end of file
+tool = { rye = { dev-dependencies = [
+    "black==22.6.0",
+    "pytest==8.1.1",
+    "pytest-cov==3.0.0",
+    "mkdocs-autorefs==0.4.1",
+    "mkdocs-material-extensions==1.0.3",
+    "mkdocs-material==8.4.3",
+    "mkdocs==1.3.1",
+    "mkdocstrings==0.19.0",
+    "Faker==13.12.1",
+    "python-dotenv>=1.0.1",
+] } }
+[project]
+name = "viadot2"
+version = "2.0a21"
+description = "A simple data ingestion library to guide data flows from some places to other places."
+authors = [{ name = "trymzet", email = "mzawadzki@dyvenia.com" }]
+dependencies = [
+    "azure-core==1.25.0",
+    "azure-storage-blob==12.13.1",
+    "awswrangler==2.19.0",
+    "s3fs==2022.11.0",
+    "boto3==1.24.59",
+    "pandas==1.4.4",
+    "pyarrow==10.0.1",
+    "pyodbc>=4.0.34, <4.1.0",
+    "openpyxl==3.0.10",
+    "jupyterlab==3.2.4",
+    "azure-identity==1.7.1",
+    "matplotlib>=3.8.3",
+    "adlfs==2022.9.1",
+    "Shapely==1.8.0",
+    "imagehash==4.2.1",
+    "visions==0.7.5",
+    "sharepy>=2.0.0, <2.1.0",
+    "simple_salesforce==1.11.5",
+    "sql-metadata==2.3.0",
+    "duckdb==0.5.1",
+    "sendgrid==6.9.7",
+    "pandas-gbq==0.19.1",
+    "pyyaml>=6.0.1",
+    "pydantic==1.10.11",
+    "aiolimiter==1.0.0",
+    "trino==0.326.*",
+    "sqlalchemy==2.0.*",
+    "minio>=7.0, <8.0",
+    "databricks-connect==11.3.*",
+]
+requires-python = ">=3.10, <3.12"
```

Comment:
Do we need such strict requirements here? Two things to note:
- this should still work on 3.9 I think?
- setting upper bounds for Python version is bad practice, should be avoided if possible. If it's really required, document why, if possible.

Reply:
[⬇️ Downgraded python version to >=3.9](https://github.com/dyvenia/viadot/pull/878/commits/f5a296d504c5a931c51d586bfc29460f5a2f49d7)

Reply:
[⬆️ Upgraded python to 3.10 version](https://github.com/dyvenia/viadot/pull/878/commits/7c1fd08c6ecff8bc9f0d139485b1893d788d7377)